tests/rest_server_test.cc

```diff
@@ -117,14 +117,14 @@ TEST(rest_server_test, basic_test)
     ASSERT_EQ(res->status, 200);
 
     // TODO: Clean this up to use proper gtest macros.
-    // NOTE: res->body is "ip6-localhost" on some architectures.
-    if (res->body == "ip6-localhost")
+    if ((res->body.length() >= 9) && // 9 being len of "localhost"
```

Comment:
instead of hardcoding 9, you could write `strlen("localhost")`. It's self-documenting

Reply:
Or, even better, you can use `EXPECT_THAT(res->body, testing::EndsWith("localhost"))`